examples/beginner-examples/token-spreader/.npmignore

```diff
@@ -0,0 +1,3 @@
+hardhat.config.js
```

Comment:
Not sure if we need an npm ignore file. This is for publishing packages to the registry.